test/api/handlers_test.go

```diff
@@ -0,0 +1,83 @@
+package api
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/skynetlabs/pinner/conf"
+	"github.com/skynetlabs/pinner/database"
+	"github.com/skynetlabs/pinner/test"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+type subtest struct {
```

Comment:
`nit` docstring

Reply:
We have been very strict about adding a docstring to every type, method and functions even when unexported because we have suffered a lot of confusion in the past (especially in testing).

For this type it's indeed pretty ridiculous to add a comment but you could also just inline it below since it appears to be the only time you are actually using it.

Either way it's not blocking this PR.

Reply:
I intend to use it in other tests in the future.
Added a docstring.